packages/account/src/Components/language-settings/language-radio-button.tsx

```diff
@@ -31,10 +32,11 @@ const LanguageRadioButton = ({ is_current_language, id, language_code, name, onC
             />
             <label htmlFor={id} className='settings-language__language--center-label'>
                 <div>
-                    <Icon
-                        icon={`IcFlag${id.replace('_', '-')}`}
-                        className='settings-language__language-link-flag settings-language__language-flag'
-                    />
+                    {TranslationFlag[language_code] ? (
+                        TranslationFlag[language_code]
+                    ) : (
+                        <Icon icon={`IcFlag${id}`} className='settings-language__language-flag' />
```

Comment:
I've remove `settings-language__language-link-flag` here because i checked in the scss there's no existing scss for it 🙏